### PR TITLE
Updated textures

### PR DESCRIPTION
**Pull request by ronnie-dumesh**
Updated filmstrips for the player's movement animations as well as setting the texture to be drawn offset from the physics body. Also added support for two different wall objects "vertical-wall" which uses the "single-brick" texture and "horizontal-wall" which uses the "double-brick" texture. These textures are being updated by Ravina. Also added support for the floor texture.

### CODE REVIEW

File: core/assets/jsons/global.json [hunk 1]
```diff
@@ -33,17 +41,27 @@
...
       "startframe": 0,
       "walklimit": 4,
-      "texture": "enemy",
+      "texture":     	{
+        "left": "enemy",
+        "right": "enemy",
+        "up": "enemy",
+        "down": "enemy",
+        "walk-left": "enemy",
+        "walk-right": "enemy"
```
Comment (BryanGraeser) [inline core/assets/jsons/global.json]: What is the difference between "left" and "walk-left" and between "right" and "walk-right"?

File: core/assets/jsons/global.json [hunk 2]
```diff
@@ -72,14 +90,27 @@
     "tint": [1.0, 1.0, 1.0, 1.0],
     "startframe":       0
   },
-  "wall": {
+  "vertical-wall": {
```
Comment (BryanGraeser) [inline core/assets/jsons/global.json]: Why the name switch between "vertical-wall" and "single-wall"? Why not just one or the other?

File: core/src/com/fallenflame/game/LevelController.java [hunk 3]
```diff
@@ -590,7 +596,9 @@ public void draw(GameCanvas canvas, float delta, BitmapFont displayFont) {
 
         // Draw all objects
         canvas.begin();
-        player.draw(canvas);
+        //TODO: draw background here
+        //canvas.draw(,0,0)
+        levelModel.draw(canvas);
```
Comment (BryanGraeser) [inline core/src/com/fallenflame/game/LevelController.java]: Why are we using level model to draw?

File: core/src/com/fallenflame/game/LightController.java [hunk 4]
```diff
@@ -211,6 +211,18 @@ public void updateLights(Collection<FlareModel> flares, Collection<EnemyModel> e
...
+        //playerLight.setPosition(player.getX() + 300f, player.getY() + 300f);
+        //Changes being made when checking using get position?
+        //System.out.println(playerLight.getPosition());
+
+        //float xOffset = l.getX() + 0.65f *  (float) Math.abs(Math.sin(o.getAngle()));
+//        System.out.println(o.getAngle());
+//        System.out.println(l.getY());
+//        float yOffset = l.getY() + 0.65f * (float) Math.abs(Math.cos(o.getAngle()));
+//        System.out.println(yOffset);
+//        System.out.println("");
```
Comment (BryanGraeser) [inline core/src/com/fallenflame/game/LightController.java]: Please delete this

**Review by BryanGraeser — CHANGES_REQUESTED** [review]
Couple minor comments, but my main question is why are we using LevelModel to draw?

File: core/assets/jsons/global.json (hunk 1 above)
Comment (ronnie-dumesh) [reply core/assets/jsons/global.json]: Was a difference originally, but deprecated now. Will remove it.

File: core/src/com/fallenflame/game/LevelController.java (hunk 3 above)
Comment (ronnie-dumesh) [reply core/src/com/fallenflame/game/LevelController.java]: Fixed

File: core/src/com/fallenflame/game/LightController.java (hunk 4 above)
Comment (ronnie-dumesh) [reply core/src/com/fallenflame/game/LightController.java]: Done

File: core/assets/jsons/global.json (hunk 2 above)
Comment (ronnie-dumesh) [reply core/assets/jsons/global.json]: Good point, will fix it

File: core/src/com/fallenflame/game/LightController.java (hunk 4 above)
Comment (ronnie-dumesh) [reply core/src/com/fallenflame/game/LightController.java]: Left the TODO for later however

**Review by BryanGraeser — APPROVED** [review]
Removed unnecessary level model changes to prevent later merge conflict. GTG now

**Pull request merged**